contribs/gmf/apps/desktop_alt/index.html

```diff
@@ -220,7 +239,7 @@
       <ngeo-modal ng-model="mainCtrl.modalShareShown" ngeo-modal-destroy-content-on-hide="true">
         <gmf-share gmf-share-email="false"/>
       </ngeo-modal>
-      <ngeo-modal ng-model="disclaimerVisibility" ngeo-modal-destroy-content-on-hide="true">
+      <ngeo-modal ng-model="XXXXdisclaimerVisibility" ngeo-modal-destroy-content-on-hide="true">
```

Comment:
Yes, this is WIP to not be bugged by the desktop_alt disclaimer popping on each F5. It will be removed before merging.